### DISCUSSION TITLE
ReelBox

### DISCUSSION THREAD

**Discussion by ua0lnj** [pbiering/ReelBoxNG#2] [General]
Hi.
I have supported reelbox plugins for vanilla vdr, the latest versions are here:
https://bitbucket.org/ua0lnj/reelbox-vanilla-vdr/src/master/

Also, you can try my fork of reelchannelscan with mcli and netseiver, I have not removed it support, but have not this hardware.
https://bitbucket.org/ua0lnj/channelscan/src/master/

If you have a 64-bit kernel driver for eHD and post it, maybe I'll update the plugins for the current vdr.

**Comment by pbiering** [comment]
Hi,

thank you very much for the links, have added them to the Wiki reference page.

My work on getting eHD kernel driver ported to 64-bit started long time ago, was not finished and I'm still looking for where I have stored the adjusted sources some years back... :-(
Potentially I have to restart work again, but the big question is: has this still a future? Potentially a survey should be made who want to use eHD still for the next years and not switching to softhddevice or similar.

**Comment by ua0lnj** [comment]
In general, I think that this is not so necessary, so I also stopped supporting ehd. But if a driver appears, can at least bring the plugins up to date.

**Reply by pbiering** [reply]
please don't forget my request for the SDK...I want to try to compile an adjusted version of "hdplayer3" and potentially also create a new boot image...goal would be to get at least a 256-color HD-ready OSD somehow working - potentially even FullHD.

**Comment by pbiering** [comment]
driver ported to 64-bit, the MIPS SDK is wanted...you have created an boot image in 2017...how?
See also: https://github.com/pbiering/ReelBoxNG/issues/4
